ddl/ddl_api.go

```diff
@@ -1469,7 +1469,6 @@ func resolveAlterTableSpec(ctx sessionctx.Context, specs []*ast.AlterTableSpec)
 			// For the compatibility, we return warning instead of error when the algorithm is COPY,
 			// because the COPY ALGORITHM is not supported in TiDB.
 			ctx.GetSessionVars().StmtCtx.AppendError(err)
-			err = nil
```

Comment:
Why remove this code? This comment said it will return a warning.

Reply:
Because this is a  ineffectually assign variable.
The ineffassign warns this variable.
`warning: ineffectual assignment to err (ineffassign)`
IMO, since this variable will be overwritted at soon , the variable is a little useless( The comment is enough ).

Reply:
Got it.